src/util/VoctCalibration.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+namespace daisy
+{
+/** @brief Helper class for calibrating an input to 1V/oct response 
+ *  @author shensley
+ * 
+ *  This provides a scale and offset value for converting
+ *  incoming CV into MIDI note numbers accurately for musical pitch tracking.
+ * 
+ *  To use, record both the 1V and 3V values using the specified functions
+ *  once calibration is complete you can use the GetData function to retrieve
+ *  the calibration values.
+ * 
+ *  This can also be used for 100mV/Semitone calibration as used by Buchla synthesizer 
+ *  modules. To calibrate for this standard. You would send 1.2V, and 3.6V
+ */
+class VoctCalibration
+{
+  public:
+    VoctCalibration() : scale_(0.f), offset_(0.f), cal_(false) {}
+
+    ~VoctCalibration() {}
+
+    bool Record(float val1V, float val3V)
+    {
+        float delta = val3V - val1V;
+        if(delta < 0.6f && delta > 0.1f)
```

Comment:
Hmm, checking the delta for valid ranges is a good idea, _but_ it's hard to quantify the expected range given hardware CV inputs can be configured a few different ways. 

If it isn't explicitly normalized by the AnalogControl class this can cause perfectly acceptable inputs to miss calibration, and fail when it should otherwise suceed.